src/olympia/lib/git.py

```diff
@@ -345,12 +345,13 @@ def extract_and_commit_from_version(cls, version, author=None, note=None):
             translation.activate('en-US')
 
             repo = cls(version.addon.id, package_type='addon')
-            file_obj = version.all_files[0]
+            file_obj = (version.all_files[0] if len(version.all_files) > 0 else
```

Comment:
`if version.all_files` is simpler and works just as well (I know, I know, I've been guilty of using `len(version.all_files)` myself in code still present in the repos :)

Reply:
oh ok, I always forget if that works in python (and I often have to think twice in JavaScript..)